client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -116,6 +134,29 @@ const MetadataEditor = createClass({
 		</div>;
 	},
 
+	renderThemeDropdown : function(){
+		const listThemes = (renderer)=>{
+			return _.map(Themes[renderer], (theme)=>{
+				return <div className='item' key={''} onClick={()=>this.handleTheme(theme)} title={''}>
+					{`${theme.renderer} : ${theme.name}`}
+				</div>;
+			});
+		};
+
+		const currentTheme = this.getThemeData(this.props.metadata.renderer, this.props.metadata.theme);
+
+		return <div className='field themes'>
+			<label>theme</label>
+			<Nav.dropdown trigger='click'>
```

Comment:
I wanted to use the nav dropdown for code reuse since the framework is all there. We can tweak the behavior for this specific instance to not disappear on mouse out.

I wanted a dropdown with a scrollbar because eventually I forsee us having 20+ options to pick from, and want to keep it from filling the whole page.

Reply:
That does make sense and it does match from a UI perspective, works for me!

Reply:
There might be accessibility issues though, but that can be addressed in its own issue for all nav dropdowns.

Reply:
> ...accessibility...

Yeah, if there's an accessibility issue with the dropdowns let's handle it separately since it would involve fixing *all* dropdowns which is unrelated to themes.